examples/custom_runbook_catalog/README.md

```
@@ -0,0 +1,89 @@
+### Option 2: Command Line
+
+```bash
+# This feature is configured via the config file
+# The catalog.json path should point to your custom catalog
+```
+
+## Directory Organization
+
+You can organize runbooks in subdirectories:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)